bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -21,23 +36,215 @@ log:set DEBUG org.openhab.core.automation
+
+// Cancel the timer.
+this.myTimer.cancel();
+
+// Check whether the timer is active. Returns true if the timer is active and will be executed as scheduled.
+let active = this.myTimer.isActive();
+
+// Reschedule the timer.
+this.myTimer.reschedule(now.plusSeconds(5));
+```
+
+### Scripts Actions
+
+Call scripts created in the UI (Settings -> Scripts) with or without parameters.
+
+```javascript
+openhab.scriptExtension = Java.type('org.openhab.core.automation.module.script.ScriptExtensionProvider');
+let bundleContext = Java.type('org.osgi.framework.FrameworkUtil').getBundle(openhab.scriptExtension.class).getBundleContext();
+openhab.RuleManager = bundleContext.getService(bundleContext.getServiceReference('org.openhab.core.automation.RuleManager'));
+
+// Simple call.
+openhab.RuleManager.runNow('<scriptToRun>');
+
+// Advanced call with arguments.
+let map = new java.util.HashMap();
+map.put('identifier1', 'value1');
+map.put('identifier2', 'value2');
+// Second argument is whether to consider the conditions, third is a Map<String, Object> (a way to pass data).
+openhab.RuleManager.runNow('<scriptToRun>', true, map);
+```
+
+Replace `<scriptToRun>` with your script's (unique-)id.
+
+## Cloud Notification Actions
+
+Notification actions may be placed in Rules to send alerts to mobile devices registered with an [openHAB Cloud instance](https://github.com/openhab/openhab-cloud) such as [myopenHAB.org](https://myopenhab.org/).
+
+For available actions have a look at the [Cloud Notification Actions Docs](https://www.openhab.org/docs/configuration/actions.html#cloud-notification-actions).
+
+```javascript
+openhab.NotificationAction = Java.type('org.openhab.io.openhabcloud.NotificationAction')
+
+// Example
+openhab.NotificationAction.sendNotification('<email>', '<message>'); // to a single myopenHAB user identified by e-mail
+openhab.NotificationAction.sendBroadcastNotification('<message>'); // to all myopenHAB users
+```
+
+Replace `<email>` with the e-mail address of the user.
+Replace `<message>` with the notification text.
+
+## Persistence Extensions
+
+For available commands have a look at [Persistence Extensions in Scripts ans Rules](https://www.openhab.org/docs/configuration/persistence.html#persistence-extensions-in-scripts-and-rules).
+
+For deeper information have a look at the [Persistence Extensions JavaDoc](https://www.openhab.org/javadoc/latest/org/openhab/core/persistence/extensions/persistenceextensions).
+
+```javascript
+openhab.PersistenceExtensions = Java.type('org.openhab.core.persistence.extensions.PersistenceExtensions');
+let ZonedDateTime = Java.type('java.time.ZonedDateTime');
+let now = ZonedDateTime.now();
+
+// Example
+var avg = openhab.PersistenceExtensions.averageSince(itemRegistry.getItem('<item>'), now.minusMinutes(5), "influxdb");
 ```
+
+Replace `<persistence>` with the persistence service to use.
+Replace `<item>` with the itemname.
+
+## Ephemeris Actions
```

Comment:
Again these are great examples and i can merge these in with the rest of the documentation that is comming